{
  "uid" : "62bf2987e3ec22b9",
  "name" : "Verify composition, visibility of elements on the 1st-6th levels of nesting in the Header\n            for an unauthorized user",
  "fullName" : "tests.header_test.TestUnauthHeaderPageStructure#test_hpu_01_02_verify_unauth_header_structure_and_visibility",
  "historyId" : "42fa829969e81ca0d1ffefbc3669d2c3",
  "time" : {
    "start" : 1728781241433,
    "stop" : 1728781241433,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55a3456b502a <unknown>\n#1 0x55a34539b5e0 <unknown>\n#2 0x55a3453937eb <unknown>\n#3 0x55a345383a09 <unknown>\n#4 0x55a345385744 <unknown>\n#5 0x55a345383cbd <unknown>\n#6 0x55a34538352f <unknown>\n#7 0x55a345383422 <unknown>\n#8 0x55a345381493 <unknown>\n#9 0x55a345381ada <unknown>\n#10 0x55a34539df41 <unknown>\n#11 0x55a34542f8c5 <unknown>\n#12 0x55a34540fb22 <unknown>\n#13 0x55a34542ed7d <unknown>\n#14 0x55a34540f8c3 <unknown>\n#15 0x55a3453dd6b3 <unknown>\n#16 0x55a3453de68e <unknown>\n#17 0x55a34567fa2b <unknown>\n#18 0x55a3456839b1 <unknown>\n#19 0x55a34566c225 <unknown>\n#20 0x55a345684532 <unknown>\n#21 0x55a34565138f <unknown>\n#22 0x55a3456a3f28 <unknown>\n#23 0x55a3456a40f3 <unknown>\n#24 0x55a3456b3e7c <unknown>\n#25 0x7fd5be894ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"a6467850bcad4dc450e7f1b04f8ffbda\")>\n\n    @pytest.fixture()\n    @allure.step(f'Open page: {MainPageLinks.URL_MAIN_PAGE}')\n    def main_page_open(driver):\n>       driver.get(MainPageLinks.URL_MAIN_PAGE)\n\ntests/conftest.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7fe0c0fcf7a0>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x55a3456a40f3 \\\\u003Cunknown>\\\\n#24 0x55a3456b3e7c \\\\u003Cunknown>\\\\n#25 0x7fd5be894ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=129.0.6668.89)\nE       Stacktrace:\nE       #0 0x55a3456b502a <unknown>\nE       #1 0x55a34539b5e0 <unknown>\nE       #2 0x55a3453937eb <unknown>\nE       #3 0x55a345383a09 <unknown>\nE       #4 0x55a345385744 <unknown>\nE       #5 0x55a345383cbd <unknown>\nE       #6 0x55a34538352f <unknown>\nE       #7 0x55a345383422 <unknown>\nE       #8 0x55a345381493 <unknown>\nE       #9 0x55a345381ada <unknown>\nE       #10 0x55a34539df41 <unknown>\nE       #11 0x55a34542f8c5 <unknown>\nE       #12 0x55a34540fb22 <unknown>\nE       #13 0x55a34542ed7d <unknown>\nE       #14 0x55a34540f8c3 <unknown>\nE       #15 0x55a3453dd6b3 <unknown>\nE       #16 0x55a3453de68e <unknown>\nE       #17 0x55a34567fa2b <unknown>\nE       #18 0x55a3456839b1 <unknown>\nE       #19 0x55a34566c225 <unknown>\nE       #20 0x55a345684532 <unknown>\nE       #21 0x55a34565138f <unknown>\nE       #22 0x55a3456a3f28 <unknown>\nE       #23 0x55a3456a40f3 <unknown>\nE       #24 0x55a3456b3e7c <unknown>\nE       #25 0x7fd5be894ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1728781182641,
      "stop" : 1728781182641,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "driver",
    "time" : {
      "start" : 1728781241433,
      "stop" : 1728781242209,
      "duration" : 776
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "main_page_open",
    "time" : {
      "start" : 1728781242209,
      "stop" : 1728781242799,
      "duration" : 590
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55a3456b502a <unknown>\n#1 0x55a34539b5e0 <unknown>\n#2 0x55a3453937eb <unknown>\n#3 0x55a345383a09 <unknown>\n#4 0x55a345385744 <unknown>\n#5 0x55a345383cbd <unknown>\n#6 0x55a34538352f <unknown>\n#7 0x55a345383422 <unknown>\n#8 0x55a345381493 <unknown>\n#9 0x55a345381ada <unknown>\n#10 0x55a34539df41 <unknown>\n#11 0x55a34542f8c5 <unknown>\n#12 0x55a34540fb22 <unknown>\n#13 0x55a34542ed7d <unknown>\n#14 0x55a34540f8c3 <unknown>\n#15 0x55a3453dd6b3 <unknown>\n#16 0x55a3453de68e <unknown>\n#17 0x55a34567fa2b <unknown>\n#18 0x55a3456839b1 <unknown>\n#19 0x55a34566c225 <unknown>\n#20 0x55a345684532 <unknown>\n#21 0x55a34565138f <unknown>\n#22 0x55a3456a3f28 <unknown>\n#23 0x55a3456a40f3 <unknown>\n#24 0x55a3456b3e7c <unknown>\n#25 0x7fd5be894ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/",
      "time" : {
        "start" : 1728781242209,
        "stop" : 1728781242799,
        "duration" : 590
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55a3456b502a <unknown>\n#1 0x55a34539b5e0 <unknown>\n#2 0x55a3453937eb <unknown>\n#3 0x55a345383a09 <unknown>\n#4 0x55a345385744 <unknown>\n#5 0x55a345383cbd <unknown>\n#6 0x55a34538352f <unknown>\n#7 0x55a345383422 <unknown>\n#8 0x55a345381493 <unknown>\n#9 0x55a345381ada <unknown>\n#10 0x55a34539df41 <unknown>\n#11 0x55a34542f8c5 <unknown>\n#12 0x55a34540fb22 <unknown>\n#13 0x55a34542ed7d <unknown>\n#14 0x55a34540f8c3 <unknown>\n#15 0x55a3453dd6b3 <unknown>\n#16 0x55a3453de68e <unknown>\n#17 0x55a34567fa2b <unknown>\n#18 0x55a3456839b1 <unknown>\n#19 0x55a34566c225 <unknown>\n#20 0x55a345684532 <unknown>\n#21 0x55a34565138f <unknown>\n#22 0x55a3456a3f28 <unknown>\n#23 0x55a3456a40f3 <unknown>\n#24 0x55a3456b3e7c <unknown>\n#25 0x7fd5be894ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"a6467850bcad4dc450e7f1b04f8ffbda\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1728781242831,
      "stop" : 1728781242893,
      "duration" : 62
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Test Header"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "header_test"
  }, {
    "name" : "subSuite",
    "value" : "TestHeaderPage"
  }, {
    "name" : "host",
    "value" : "fv-az585-604"
  }, {
    "name" : "thread",
    "value" : "2289-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.header_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 3,
        "skipped" : 0,
        "passed" : 79,
        "unknown" : 0,
        "total" : 82
      },
      "items" : [ {
        "uid" : "6df33b958909b96e",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/150//#testresult/6df33b958909b96e",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55d6c4a1802a <unknown>\n#1 0x55d6c46fe5e0 <unknown>\n#2 0x55d6c46f67eb <unknown>\n#3 0x55d6c46e6a09 <unknown>\n#4 0x55d6c46e8744 <unknown>\n#5 0x55d6c46e6cbd <unknown>\n#6 0x55d6c46e652f <unknown>\n#7 0x55d6c46e6422 <unknown>\n#8 0x55d6c46e4493 <unknown>\n#9 0x55d6c46e4ada <unknown>\n#10 0x55d6c4700f41 <unknown>\n#11 0x55d6c47928c5 <unknown>\n#12 0x55d6c4772b22 <unknown>\n#13 0x55d6c4791d7d <unknown>\n#14 0x55d6c47728c3 <unknown>\n#15 0x55d6c47406b3 <unknown>\n#16 0x55d6c474168e <unknown>\n#17 0x55d6c49e2a2b <unknown>\n#18 0x55d6c49e69b1 <unknown>\n#19 0x55d6c49cf225 <unknown>\n#20 0x55d6c49e7532 <unknown>\n#21 0x55d6c49b438f <unknown>\n#22 0x55d6c4a06f28 <unknown>\n#23 0x55d6c4a070f3 <unknown>\n#24 0x55d6c4a16e7c <unknown>\n#25 0x7fa8b7494ac3 <unknown>",
        "time" : {
          "start" : 1728694564754,
          "stop" : 1728694564754,
          "duration" : 0
        }
      }, {
        "uid" : "252ac9dacdfc7558",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/149//#testresult/252ac9dacdfc7558",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55b8c99dd02a <unknown>\n#1 0x55b8c96c35e0 <unknown>\n#2 0x55b8c96bb7eb <unknown>\n#3 0x55b8c96aba09 <unknown>\n#4 0x55b8c96ad744 <unknown>\n#5 0x55b8c96abcbd <unknown>\n#6 0x55b8c96ab52f <unknown>\n#7 0x55b8c96ab422 <unknown>\n#8 0x55b8c96a9493 <unknown>\n#9 0x55b8c96a9ada <unknown>\n#10 0x55b8c96c5f41 <unknown>\n#11 0x55b8c97578c5 <unknown>\n#12 0x55b8c9737b22 <unknown>\n#13 0x55b8c9756d7d <unknown>\n#14 0x55b8c97378c3 <unknown>\n#15 0x55b8c97056b3 <unknown>\n#16 0x55b8c970668e <unknown>\n#17 0x55b8c99a7a2b <unknown>\n#18 0x55b8c99ab9b1 <unknown>\n#19 0x55b8c9994225 <unknown>\n#20 0x55b8c99ac532 <unknown>\n#21 0x55b8c997938f <unknown>\n#22 0x55b8c99cbf28 <unknown>\n#23 0x55b8c99cc0f3 <unknown>\n#24 0x55b8c99dbe7c <unknown>\n#25 0x7fe54fe94ac3 <unknown>",
        "time" : {
          "start" : 1728608178543,
          "stop" : 1728608178543,
          "duration" : 0
        }
      }, {
        "uid" : "dc269c2a2816ca78",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/322//#testresult/dc269c2a2816ca78",
        "status" : "passed",
        "time" : {
          "start" : 1728556926945,
          "stop" : 1728556937173,
          "duration" : 10228
        }
      }, {
        "uid" : "6e4377c75bc7d469",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/321//#testresult/6e4377c75bc7d469",
        "status" : "passed",
        "time" : {
          "start" : 1728553847976,
          "stop" : 1728553858212,
          "duration" : 10236
        }
      }, {
        "uid" : "f1c3a6b5093a3549",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/148//#testresult/f1c3a6b5093a3549",
        "status" : "passed",
        "time" : {
          "start" : 1728521948044,
          "stop" : 1728521958309,
          "duration" : 10265
        }
      }, {
        "uid" : "c7df55f931f3435d",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/320//#testresult/c7df55f931f3435d",
        "status" : "passed",
        "time" : {
          "start" : 1728470403468,
          "stop" : 1728470413728,
          "duration" : 10260
        }
      }, {
        "uid" : "130df4d513655d6e",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/319//#testresult/130df4d513655d6e",
        "status" : "passed",
        "time" : {
          "start" : 1728467815775,
          "stop" : 1728467826050,
          "duration" : 10275
        }
      }, {
        "uid" : "95008a934a9adc2",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/147//#testresult/95008a934a9adc2",
        "status" : "passed",
        "time" : {
          "start" : 1728435665580,
          "stop" : 1728435675816,
          "duration" : 10236
        }
      }, {
        "uid" : "fc192f719e985d9d",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/318//#testresult/fc192f719e985d9d",
        "status" : "passed",
        "time" : {
          "start" : 1728377923415,
          "stop" : 1728377933633,
          "duration" : 10218
        }
      }, {
        "uid" : "78a07d48a741c06c",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/317//#testresult/78a07d48a741c06c",
        "status" : "passed",
        "time" : {
          "start" : 1728375126383,
          "stop" : 1728375136602,
          "duration" : 10219
        }
      }, {
        "uid" : "2fe003369d201b72",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/316//#testresult/2fe003369d201b72",
        "status" : "passed",
        "time" : {
          "start" : 1728375012839,
          "stop" : 1728375023114,
          "duration" : 10275
        }
      }, {
        "uid" : "f662bc225ea0b06",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/146//#testresult/f662bc225ea0b06",
        "status" : "passed",
        "time" : {
          "start" : 1728349093482,
          "stop" : 1728349103698,
          "duration" : 10216
        }
      }, {
        "uid" : "28d09e99de11d841",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/315//#testresult/28d09e99de11d841",
        "status" : "passed",
        "time" : {
          "start" : 1728292672504,
          "stop" : 1728292682721,
          "duration" : 10217
        }
      }, {
        "uid" : "1b1c86319edcc896",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/314//#testresult/1b1c86319edcc896",
        "status" : "passed",
        "time" : {
          "start" : 1728289716765,
          "stop" : 1728289727009,
          "duration" : 10244
        }
      }, {
        "uid" : "5bc1c1b5f14fea6b",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/145//#testresult/5bc1c1b5f14fea6b",
        "status" : "passed",
        "time" : {
          "start" : 1728263123842,
          "stop" : 1728263134056,
          "duration" : 10214
        }
      }, {
        "uid" : "20cc97329b51f372",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/313//#testresult/20cc97329b51f372",
        "status" : "passed",
        "time" : {
          "start" : 1728226242950,
          "stop" : 1728226253165,
          "duration" : 10215
        }
      }, {
        "uid" : "45ef90224622d17e",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/312//#testresult/45ef90224622d17e",
        "status" : "passed",
        "time" : {
          "start" : 1728221379239,
          "stop" : 1728221389460,
          "duration" : 10221
        }
      }, {
        "uid" : "67b7e565b0076f23",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/144//#testresult/67b7e565b0076f23",
        "status" : "passed",
        "time" : {
          "start" : 1728176654141,
          "stop" : 1728176664362,
          "duration" : 10221
        }
      }, {
        "uid" : "267c71d165b2c6aa",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/311//#testresult/267c71d165b2c6aa",
        "status" : "passed",
        "time" : {
          "start" : 1728117324120,
          "stop" : 1728117334346,
          "duration" : 10226
        }
      }, {
        "uid" : "2480d15db1420e3a",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/310//#testresult/2480d15db1420e3a",
        "status" : "passed",
        "time" : {
          "start" : 1728108720144,
          "stop" : 1728108730363,
          "duration" : 10219
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "62bf2987e3ec22b9.json",
  "parameterValues" : [ ]
}